api/routers/legacy.py

```diff
@@ -750,9 +763,12 @@ async def get_contributor_id(contributor: str):
 
     if player:
         return_dict = {
-            "id": player.id
+            "id": player[0].id #This might suck for a bit until duplicate norm_names are removed from the DB
```

Comment:
It's because of the duplicates. Since we query by normalized_name we can end up with multiple accounts from get_player. I didn't add anything to determine which one is actually the one we use. This will break this route for my account, for example, because "Seltzer_Bro" has a lower id than the other entry I have that is actually in use. The underlying issue is the duplicates, and that may be better to tackle first. Then 90% of this PR can be trashed lol